src/pybind/mgr/dashboard/run-frontend-unittests.sh

```diff
@@ -31,7 +31,7 @@ if [ $? -gt 0 ]; then
   echo -e "\nTranslations extraction has failed."
 else
   i18n_lint=`awk '/<source> |<source>$| <\/source>/,/<\/context-group>/ {printf "%-4s ", NR; print}' src/locale/messages.xlf`
-  if [ "$i18n_lint" ]; then
+  if [ "$i18n_lint" ] && [[ $i18n_lint != *"node_modules/"* ]]; then
```

Comment:
IMHO the conditional checks can be simplified .

Reply:
Done!